%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Wagner CV
% XeLaTeX Template
%
% The original file was downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Howard Wilson (https://github.com/watsonbox/cv_template_2004) with
% extensive modifications by Vel (vel@latextemplates.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[10pt]{article} % Default font size

\input{structure.tex} % Include the file specifying document layout

%----------------------------------------------------------------------------------------

\begin{document}

%----------------------------------------------------------------------------------------
%	NAME AND CONTACT INFORMATION
%----------------------------------------------------------------------------------------

\title{Alex Handler Wagner, Ph.D. -- Curriculum Vitae} % Print the main header

%------------------------------------------------

\parbox{0.5\textwidth}{ % First block
\begin{tabbing} % Enables tabbing
\hspace{3cm} \= \hspace{4cm} \= \kill % Spacing within the block
{\bf Address} \> 575 Children's Crossroad,\\ % Address line 1
\> Columbus, OH 43215 \\ % Address line 2
{\bf Date of Birth} \> 16$^{th}$ June 1985 % Date of birth 
\end{tabbing}}
\hfill % Horizontal space between the two blocks
\parbox{0.5\textwidth}{ % Second block
\begin{tabbing} % Enables tabbing
\hspace{3cm} \= \hspace{4cm} \= \kill % Spacing within the block
{\bf Work Phone} \> +1 (614) 355 1645 \\ % Home phone
{\bf Email} \> \href{mailto:Alex.Wagner@nationwidechildrens.org}{Alex.Wagner@nationwidechildrens.org} \\ % Email address
{\bf Social} \> \href{https://genomic.social/@a}{genomic.social/@a}
 
\end{tabbing}}

%----------------------------------------------------------------------------------------
%	PERSONAL PROFILE
%----------------------------------------------------------------------------------------

\section{Personal Profile}

My ongoing work is to integrate genomic evidence into knowledge used for interpreting genomic variation. My research program is defined by the development of several precision medicine web tools, for aggregating and disseminating genomic knowledge particularly in the cancer domain. These include the \href{www.dgidb.org}{Drug-Gene Interaction Database}, the database of Clinical Interpretations of Variants in Cancer
(\href{https://www.civicdb.org}{CIViC}) knowledgebase, and the \href{https://search.cancervariants.org}{VICC meta-knowledgebase}. My community science research program requires active engagement in professional consortia, and I currently hold advisory or leadership roles in several organizations, including the Variant Interpretation for Cancer Consortium (VICC) (as Director), the Global Alliance for Genomics and Health (\href{https://ga4gh.org}{GA4GH}) (steering committee), the Atlas for Variant Effects Alliance (CVI and DCD committees), and ClinGen (Somatic Cancer, Cancer Variant Interpretation, and Data Platform work groups). I also co-led the development of the GA4GH \href{https://vrs.ga4gh.org}{Variation Representation Specification}, and led the 2021 manuscript first describing that work. Ongoing support for this research is provided by a Genomic Innovator Award
from the National Human Genome Research Institute.

%----------------------------------------------------------------------------------------
%	RESEARCH EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\section{Research Experience}

\compjob
{Aug 2020 - }{Present}
{Institute for Genomic Medicine, Nationwide Children's Hospital}
{https://www.nationwidechildrens.org/specialties/institute-for-genomic-medicine}
{Principal Investigator}

%------------------------------------------------

\compjob
{Aug 2020 - }{Present}
{Departments of Pediatrics and Biomedical Informatics, the Ohio State University}
{https://medicine.osu.edu/departments}
{Assistant Professor}

%------------------------------------------------

\job
{Jun 2019 -}{Jul 2020}
{Division of Oncology, Washington University in St. Louis}
{https://oncology.wustl.edu/}
{Instructor}
{Advisors: Obi L. Griffith, Ph.D.; Timothy J. Ley, M.D.}

%------------------------------------------------

\job
{Jan 2015 -}{May 2019}
{McDonnell Genome Institute, Washington University in St. Louis}
{https://genome.wustl.edu/}
{Postdoctoral Research Scholar}
{Advisors: Obi L. Griffith, Ph.D.; Ramaswamy Govindan, M.D.}

%------------------------------------------------

\job
{Aug 2010 -}{Dec 2014}
{Coordinated Laboratory for Computational Genomics, University of Iowa}
{https://clcg.lab.uiowa.edu/}
{Graduate Research Assistant}
{Advisors: Terry A. Braun, Ph.D.; Edwin M. Stone, M.D., Ph.D.}

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------

\section{Education}

\edu
{2010-2014}
{Ph.D, Computational Genetics}
{University of Iowa}
{http://genetics.grad.uiowa.edu/about-our-program/computational-genetics}
{3.90}

%------------------------------------------------

\edu
{2010-2013}
{Graduate Certificate, Bioinformatics}
{University of Iowa}
{https://informatics.uiowa.edu/study-opportunities/graduate-program/bioinformatics}
{3.96}

%------------------------------------------------

\edu
{2010-2013}
{BS, Biology (Minor: Mathematics)}
{Iowa State University}
{https://www.biology.iastate.edu/}
{3.51  - \textit{Cum Laude}}

%----------------------------------------------------------------------------------------
%	PUBLISHED WORKS
%----------------------------------------------------------------------------------------

\section{Peer-reviewed Publications}

*Denotes Co-First Authorship

\begin{enumerate}
 
\input{pub.tex}

\end{enumerate}

\section{Other Published Works}

\begin{enumerate}

\input{otherPub.tex}

\end{enumerate}

%----------------------------------------------------------------------------------------
%	FUNDING SECTION
%----------------------------------------------------------------------------------------

\section{Funding: Research and Fellowship Grant Awards}

\nihFund
{NIH NHGRI U24 Competitive Subaward - GREGoR Consortium}
{2024}{Present}
{U24HG011746}{10216104}
{Subaward PI}

\nihFund
{NIH NCI U24 - ClinGen Somatic Biomedical Knowledgebase}
{2023}{Present}
{U24CA275783}{10555024}
{Co-I}

\nihFund
{NIH OD U54 - Bridge2AI BRIDGE Center (Standards)}
{2022}{Present}
{U54HG012513}{10661023}
{Standards Core co-Lead}

\nihFund
{NIH NHGRI R35 Supplement - AI-Ready gnomAD}
{2022}{2023}
{R35HG011949}{10594234}
{PI}

\nihFund
{NIH NHGRI R35 - Genomic Innovator}
{2021}{Present}
{R35HG011949}{10485282}
{PI}

\nihFund
{NIH NHGRI K99/R00 - Knowledge Harmonization}
{2019}{Present}
{R35HG011949}{10485282}
{PI}

\nihFund
{NIH NCI F32 - Cancer Genomics}
{2017}{2019}
{R35HG011949}{10485282}
{PI}

\section{Funding: Training Awards}

\nihFund
{NIH NCI T32 Postdoctoral Training - Cancer Biology}
{2016}{2017}
{T32CA009547}{9073871}
{Trainee}

\nihFund
{NIH NCI T32 Predoctoral Training - Genetics}
{2013}{2014}
{T32GM008629}{8685989}
{Trainee}

\nihFund
{NIH NCI T32 Predoctoral Training - Bioinformatics}
{2011}{2013}
{T32GM082729}{8501534}
{Trainee}

%----------------------------------------------------------------------------------------
%	MERIT SECTION
%----------------------------------------------------------------------------------------

\section{Research Recognition Awards}

\merit
{Genomic Innovator Award}
{https://www.genome.gov/research-funding/Funding-Opportunities/Genomic-Innovator-Awards}
{National Human Genome Research Institute, National Institutes of Health}
{2021}
{National recognition awarded for early career success in consortia-driven scientific research.}

\merit
{ICTS Precision Medicine Abstract Award}
{http://griffithlab.org/WagnerICTS}
{Washington University in St. Louis}
{2018}
{The most outstanding research was selected from more than 75 applicants at the Washington University Precision Medicine Symposium to present to Eric Green, the director of the National Human Genome Research Institute (NHGRI).}

\merit
{D.C. Spriestersbach Dissertation Prize Nominee}
{https://grad.uiowa.edu/faculty-staff/dgs-graduate-faculty/awards/spriestersbach-prize}
{University of Iowa, Genetics Program Nomination}
{2015}
{This biennial award recognizes excellence in doctoral research. Each of the twenty biological/life sciences programs at the University of Iowa nominates one dissertation submitted between July 1, 2013 and June 30, 2015 to compete for the award.}

\merit
{Outstanding Student Research Award - Bioinformatics and Computational Biology}
{https://engineering.uiowa.edu/faculty-and-staff/excellence-awards}
{University of Iowa, College of Engineering}
{2013}
{This annual award recognizes a single student in the College of Engineering for exemplary research in the fields of bioinformatics and computational biology.}

\section{Travel and Trainee Awards}

\travelAward
{Trainee Abstract Award}
{Curating the Clinical Genome Annual Conference}
{Washington D.C.}
{2019}

\travelAward
{1st Place Student/Trainee Travel Award}
{Cancer Genomics Consortium Annual Conference}
{Nashville, TN}
{2018}

\travelAward
{AACR-Bristol Myers Squibb Scholar-in-Training Award}
{American Association for Cancer Research}
{Chicago, IL}
{2018}

\travelAward
{NSF Travel Grant}
{Intelligent Systems for Molecular Biology Annual Conference}
{Berlin, Germany}
{2013}

\travelAward
{Graduate Student Senate Travel Grant}
{Association for Research in Vision and Ophthalmology Annual Conference}
{Seattle, WA}
{2013}

% From prior CV; can add in at later time

%\textbf{Oral Presentation Awards}
%\begin{innerlist}
%\item \href{https://genome.uiowa.edu/home/node/72}{Annual Bioinformatics Retreat, University of Iowa}\hfill August 16, 2013
%	\begin{innerlist}
%	\item \emph{Best Student Talk}
%	\end{innerlist}
%\item \href{http://webeye.ophth.uiowa.edu/eig/MERS_2012.html}{Midwest Eye Research Symposium}\hfill July 6, 2012
%	\begin{innerlist}
%	\item \emph{Outstanding Oral Presentation}, 2nd Place
%	\end{innerlist}
%\end{innerlist}
%
%\halfblankline

%\halfblankline
%
%\textbf{Poster Presentation Awards}
%\begin{innerlist}
%\item \href{http://ppc.uiowa.edu/health/ihg/poster}{Interdisciplinary Health Research Poster Session}\hfill April 23, 2013
%	\begin{innerlist}
%	\item \emph{Best Poster Award}, Center on Aging
%	\end{innerlist}
%\end{innerlist}

%----------------------------------------------------------------------------------------
%	PRESENTATIONS SECTION
%----------------------------------------------------------------------------------------

\section{Presentations}

\input{presentations.tex}

%----------------------------------------------------------------------------------------
%	INSTRUCTION SECTION
%----------------------------------------------------------------------------------------

\section{Instruction}

\instruction
{Genomic Knowledge Standards 2-day Secret Agent Workshop}
{Workshop Organizer}
{Apr 2024}{}
{GA4GH Connect}
{Ascona, Switzerland}

\instruction
{CV and Cover Letter Preparation - Professional Development Seminar Series}
{Panelist}
{Feb 2024}{}
{Cancer Genomics Consortium}
{Virtual}

\instruction
{IGM Trainee Writing Workshop}
{Workshop Instructor}
{2023 -}{Present}
{Institute for Genomic Medicine}
{Columbus, OH}

\instruction
{VICC/ClinGen/CIViC Hackathon}
{Workshop Organizer}
{Aug 2023}{}
{CGC Annual Meeting}
{St. Louis, MO}

\instruction
{The GA4GH Variation Representation Specification Hackathon}
{Workshop Organizer}
{Sep 2022}{}
{GA4GH Plenary 2022}
{Barcelona, Spain}

\instruction
{CGC Bioinformatics Workshop}
{Workshop Moderator}
{Aug 2022}{}
{Cancer Genomics Consortium Annual Meeting}
{St. Louis, MO}

\instruction
{Genomic Knowledge Standards 2-day Spaghetti Western Workshop}
{Workshop Moderator}
{Apr 2022}{}
{GA4GH Connect 2022}
{Montreal, Canada}

\instruction
{Clinical Genomics Career Panel}
{Panelist}
{Jul 2021}{}
{ClinGen}
{Virtual}

\instruction
{Variation Representation Specification (VRS) Workshop}
{Workshop Organizer}
{Jun 2021}{}
{Genomics in Health Implementation Forum}
{Virtual}

\instruction
{Harmonizing Variant Interpretation - Advanced Course on Biomarkers in Precision Medicine}
{Instructor}
{Jun 2021}{}
{ESMO Asia}
{Singapore}

\instruction
{Variant Interpretation}
{Guest Lecturer}
{Dec 2019}{}
{University of Zurich}
{Zurich, Switzerland}

\instruction
{Bioinformatics: Understanding what's Underneath the Hood}
{Guest Lecturer}
{Aug 2019}{}
{Cancer Genomics Consortium Annual Conference}
{Nashville, TN}

\instruction
{Genomic Knowledge Standards}
{Workshop Leader}
{Mar 2019}{}
{AMED-GA4GH Workshop}
{Sendai, Japan}

\instruction
{Advanced Sequencing Technologies and Applications}
{Teaching Assistant \& Lecturer}
{Nov 2018}{}
{Cold Spring Harbor Laboratories}
{Cold Spring Harbor, NY}

\instruction
{Introduction to bioinformatics learning resources}
{Workshop Instructor}
{Aug 2018}{}
{Cancer Genomics Consortium Annual Conference}
{Nashville, TN}

\instruction
{Escape from Perlgatory: Developing in Python and Ruby}
{Workshop Instructor}
{Aug 2017 -}{Jun 2019}
{McDonnell Genome Institute}
{Saint Louis, MO}

\instruction
{Advanced Sequencing Technologies and Applications}
{Teaching Assistant \& Lecturer}
{Nov 2017}{}
{Cold Spring Harbor Laboratories}
{Cold Spring Harbor, NY}

\instruction
{CIViC Hackathon}
{Workshop Instructor}
{Nov 2016}{}
{Netherlands Cancer Institute (NKI)}
{Amsterdam, Netherlands}

\instruction
{Advanced Sequencing Technologies and Applications}
{Teaching Assistant \& Lecturer}
{Nov 2016}{}
{Cold Spring Harbor Laboratories}
{Cold Spring Harbor, NY}

\instruction
{Bioinformatics Techniques}
{Teaching Assistant}
{Aug 2014 -}{Dec 2014}
{Department of Biomedical Engineering, University of Iowa}
{Iowa City, IA}

\instruction
{Bioimaging and Bioinformatics}
{Teaching Assistant}
{Jan 2014 -}{May 2014}
{Department of Biomedical Engineering, University of Iowa}
{Iowa City, IA}

\instruction
{Computational Genomics}
{Teaching Assistant}
{Jan 2014 -}{May 2014}
{Department of Biomedical Engineering, University of Iowa}
{Iowa City, IA}

\instruction
{Bioinformatics Techniques}
{Teaching Assistant}
{Aug 2013 -}{Dec 2013}
{Department of Biomedical Engineering, University of Iowa}
{Iowa City, IA}

\instruction
{Introduction to Bioinformatics Computing with Python}
{Instructor}
{Aug 2013 -}{Dec 2013}
{Department of Biomedical Engineering, University of Iowa}
{Iowa City, IA}

%\item[] BIOL 313 - Principles of Genetics\\
%	Instructor: Jack Girton\\
%	Department of Biology\\
%	Iowa State University
%\end{innerlist}
%\end{samepage}

%----------------------------------------------------------------------------------------
%	SERVICE SECTION
%----------------------------------------------------------------------------------------

\section{Service}

\cmte
{HGVS Variation Nomenclature Committee (HVNC)}
{Elected Member}
{2022}{Present}
{The HGVS Variation Nomenclature Committee (HVNC) is responsible for the development
and maintenance of the HGVS nomenclature for describing sequence variation.
Members are elected for 4 year terms.}

 \cmte
{Cancer Genomics Consortium Annual Meeting Program Committee}
{Member}
{2022}{2024}
{This committee sets the program for the annual meeting of the Cancer Genomics Consortium. I have served on this
committee for 2 subsequent years: 2022-23 and 2023-24.}

\cmte
{GA4GH Steering Committee}
{Driver Project Champion (VICC)}
{2019}{Present}
{The GA4GH Steering Committee is responsible for reviewing and voting on procedural motions
regarding the development of global data standards. The steering committee consists of leads
of technical work streams as well as national and international Driver Projects.}

\cmte
{Variation Representation}
{Appointed Lead}
{2019}{Present}
{The Variation Representation group of the Genomic Knowledge Standards Work Stream is
 responsible for developing and maintaining standards for the representation of variation as used
 in genomic knowledgebases. This position is by appointment of the Work Stream leads.}

\cmte
{Variant Interpretation for Cancer Consortium}
{Director}
{2019}{Present}
{The Variant Interpretation for Cancer Consortium works to standardize the curation, 
representation, and interpretation of clinically-relevant evidence associated with genomic variation
 in cancers. VICC Directors are responsible for guiding the organization through policy, outreach, and
 oversight. New directors are appointed by sitting directors.}
 
 \cmte
{Clinical Interpretation of Variants in Cancer (CIViC) knowledgebase}
{Editor}
{2015}{Present}
{The CIViC knowledgebase is used to curate information about genetic variants and their role in 
informing clinical interpretation of cancers. Platform editors are content and platform experts that 
review and approve submissions by community members. I have moderated over 450 entries on
the CIViC platform.}

 \cmte
{ClinGen Committees – Data Platform, PGx, and CVI}
{Appointed Member}
{2020}{Present}
{ClinGen committees are groups of experts that are invited to participate in a particular domain.
Data Platform is a team focused on genomic knowledge interoperability, Pharmacogenomics (PGx) is
a team focused on pharmacogenomic knowledge standardization, and Cancer Variant Interpretation (CVI)
focuses on the development of rules for use by Somatic Cancer Variant Curation Expert Panels
(SC-VCEPs).}

 \cmte
{Atlas of Variant Effects Alliance (AVE) Committees – CVI and DCD}
{Appointed Member}
{2019}{Present}
{AVE committees are groups of experts that are invited to participate in a particular domain.
Clinical Variant Interpretation (CVI) is a committee focused on applying MAVE data to clinical interpretation,
Data Coordination and Dissemination (DCD) is a committee focused on MAVE data standardization and sharing.}

 \cmte
{ASH Precision Medicine Somatic Working Group}
{Appointed Member}
{2020}{2022}
{The ASH Precision Medicine Working Group focused on the integration of variant interpretation data for
hematological malignancies from different institutions.}


%\parbox{0.5\textwidth}{ % First block
%\begin{tabbing}
%\hspace{2.75cm} \= \hspace{4cm} \= \kill % Spacing within the block
%{\bf Name} \> Bill Lumbergh \\ % Referee name
%{\bf Company} \> Initech Inc. \\ % Referee company
%{\bf Position} \> Vice President \\ % Referee job title 
%{\bf Contact} \> \href{mailto:bill@initech.com}{bill@initech.com} % Referee contact information
%\end{tabbing}}
%\hfill % Horizontal space between the two blocks
%\parbox{0.5\textwidth}{ % Second block
%\begin{tabbing}
%\hspace{2.75cm} \= \hspace{4cm} \= \kill % Spacing within the block
%{\bf Name} \> Michael "Big Mike" Tucker\\ % Referee name
%{\bf Company} \> Burbank Buy More \\ % Referee company
%{\bf Position} \> Store Manager \\ % Referee job title 
%{\bf Contact} \> \href{mailto:mike@buymore.com}{mike@buymore.com} % Referee contact information
%\end{tabbing}}

%----------------------------------------------------------------------------------------

\end{document}